msprime/trees.py

```diff
@@ -676,12 +658,10 @@ def draw(
         :param bool show_mutation_labels: If True, show labels for mutations.
         :param bool show_internal_node_labels: If True, show labels for internal nodes.
         :param bool show_leaf_node_labels: If True, show labels for leaf nodes.
-        :param list branch_colours: A list giving colours for edges in the tree.
-            If this is an array such as ['red','blue'], the indexes of the array
-            correspond to the node whose edges are coloured. If a dict, the keys
-            give the node numbers for each colour. Unspecified edges are coloured  
-            according to the consensus colour of their children, or use the default
-            colour if there is disagreement.
+        :param list branch_colours: A dict giving colours for edges in the tree.
+            Keys correspond to node numbers in the tree. 
```

Comment:
Need to explain which node here (i.e., the child or parent node). Also, the param type is incorrect, should be 'dict'.